docs/platforms/macos.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'macOS'
+---
+
+# macOS builds
+
+## Pre-requisites
+
+Pre-requisite: you need to have native build tools installed.
+
+Because the builds are happening without full isolation, there might be some differences compared to CI builds (Xcode version, OS version, local files, ...) that might prevent you from finding an issue only seen in CI.
+
+In order to speed-up builds, cibuildwheel will cache the tools it needs to be reused for future builds. The folder used for caching is system/user dependent and is reported in the printed preamble of each run (e.g. `Cache folder: /Users/Matt/Library/Caches/cibuildwheel`).
+
+You can override the cache folder using the `CIBW_CACHE_PATH` environment variable.
+
+!!! warning
+    cibuildwheel uses official python.org macOS installers for CPython but those can only be installed globally.
+
+    In order not to mess with your system, cibuildwheel won't install those if they are missing. Instead, it will error out with a message to let you install th missing CPython:
```

Comment:
```suggestion
    In order not to mess with your system, cibuildwheel won't install those if they are missing. Instead, it will error out with a message to let you install the missing CPython:
```

Reply:
Resolved.